exporters/metrics/src/main/java/com.google.cloud.opentelemetry.metric/MetricExporter.java

```diff
@@ -0,0 +1,205 @@
+package com.google.cloud.opentelemetry.metric;
+
+import static com.google.api.client.util.Preconditions.checkNotNull;
+import static com.google.cloud.opentelemetry.metric.MetricTranslator.mapGcpMonitoredResource;
+import static com.google.cloud.opentelemetry.metric.MetricTranslator.mapMetric;
+import static com.google.cloud.opentelemetry.metric.MetricTranslator.mapMetricDescriptor;
+import static com.google.cloud.opentelemetry.metric.MetricTranslator.mapPoint;
+import static java.util.logging.Level.WARNING;
+
+import com.google.api.Metric;
+import com.google.api.MetricDescriptor;
+import com.google.api.gax.core.FixedCredentialsProvider;
+import com.google.auth.Credentials;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.monitoring.v3.MetricServiceClient;
+import com.google.cloud.monitoring.v3.MetricServiceSettings;
+import com.google.cloud.monitoring.v3.stub.MetricServiceStub;
+import com.google.common.collect.Lists;
+import com.google.monitoring.v3.CreateMetricDescriptorRequest;
+import com.google.monitoring.v3.Point;
+import com.google.monitoring.v3.ProjectName;
+import com.google.monitoring.v3.TimeSeries;
+import io.opentelemetry.common.Labels;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import java.io.IOException;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.logging.Logger;
+
+public class MetricExporter implements io.opentelemetry.sdk.metrics.export.MetricExporter {
+
+  private static final String PROJECT_NAME_PREFIX = "projects/";
+  private static final long WRITE_INTERVAL_SECOND = 10;
```

Comment:
In some recent conversations it has been recommended to use 12 seconds as a timeout for Cloud Monitoring. Could you please change to that value?